protos/bottle/assembly/v1.proto

```diff
@@ -5,5 +5,5 @@ import "bottle/assembly/v1/service.proto";
 package bottle.assembly;
 
 service V1 {
-  rpc BuildList(bottle.assembly.v1.BuildListRequest) returns (bottle.assembly.v1.BuildListResponse);
+  rpc BuildList(bottle.assembly.v1.BuildListRequest) returns (stream bottle.assembly.v1.BuildResponse);
```

Comment:
Thoughts on this change @btkostner? Since we're streaming back the builds I don't think collection makes sense in the reply.

Reply:
That works, and will probably cut down on a lot of obscure list messages we would make down the line